services/ingestion/subscriber.go

```diff
@@ -252,3 +261,47 @@ func (r *RPCSubscriber) blocksFilter() flow.EventFilter {
 		},
 	}
 }
+
+func (r *RPCSubscriber) fetchBlockEvents(
```

Comment:
add a comment explaining what this function does and how it is used for backup

Reply:
Good point :+1: 
Added in https://github.com/onflow/flow-evm-gateway/pull/529/commits/76d544c7e08a528858e034b4e99bf965a4361830